index.html

```diff
@@ -10,8 +10,13 @@
     <link rel="stylesheet" type="text/css" href="styles/main.css">
 </head>
 <body __wid="TDi4iuCp">
-<div id="d3_container" __wid="iZUBjGJV">
-    <svg class="canvas" id="canvas" __wid="PDgACC8e"></svg>
-</div>
+    <div id="CogCanvas">
```

Comment:
I think "svg" canvas and "content_container" can belong to a same flex box. Do we still need "div" CogCanvas.

Reply:
Left Undone For future work.